server/src/models/channel.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @todo: Rename table to "channels".
+ * @param sequelize
+ * @param DataTypes
+ * @return {*}
+ */
+module.exports = (sequelize, DataTypes) => {
+  const Channel = sequelize.define('Channel', {
+    id: {
+      type: DataTypes.UUID(),
+      primaryKey: true,
+      defaultValue: sequelize.fn('uuid_generate_v4'),
+      allowNull: false,
+    },
+    name: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      unique: true,
+    },
+    description: {
+      type: DataTypes.TEXT,
+      allowNull: true,
+    },
+    isArchived: {
+      type: DataTypes.BOOLEAN,
+      allowNull: false,
+      default: false,
+    },
+    isPublic: {
+      type: DataTypes.BOOLEAN,
+      allowNull: false,
+      default: true,
+    },
+  });
+
+
+  Channel.associate = function (models) {
+    Channel.Creator = Channel.belongsTo(models.Agent, {
+      foreignKey: 'creatorId',
+      as: 'creator'
+    });
```

Comment:
Can you also add ``Chanel.Agents`` or similar, that lists all members, via ``AgentChannel``, then do the same for an agent (``Agent.Channels``)

Reply:
This could also be part of a different story, if there's another specific one in mind.